src/modules/SdnDiag.NetworkController.psm1

```diff
@@ -321,13 +321,20 @@ function Get-SdnClusterType {
     )
 
     $sb = {
-        # with failover cluster, the SDNApiService will run as a service within windows
-        # so we can check if the service exists to determine if it is a failover cluster configuration regardless if running
-        $service = Get-Service -Name 'SDNApiService' -ErrorAction Ignore
-        if ($service) {
+        # 1. First check if ClusSvc service exists
+        $clusService = Get-Service -Name 'ClusSvc' -ErrorAction Ignore
```

Comment:
@luyaof curious on this one. In my lab environments for testing and what I am seeing with validatgion testing, is the service 'SdnApiService' exists, it's just not running unless it's the primary owner node. Are you seeing something different in your environments?

Reply:
I see the issue. My lab is running on an old OS build. There was a change of the service name with prefix "SDN" added. So my lab showing service name `ApiService` instead of `SdnApiService` so failed detect the cluster type.